USER_MANUAL.md

```diff
@@ -639,6 +639,18 @@ See the following files for examples on setting this value at a more granular le
 
 **Notes of caution:** N/A
 
+#### &lt;Pre-Simulation Emissions&gt;
+
+**Data Type:** Boolean
+
+**Default input:** None
+
+**Description:** This parameter specifies if LDAR-Sim should generate emissions existing before the start date of the simulation. If "True", emissions beginning up to duration days before the start of the simulation will be generated. This essentially initializes the system from a state of no LDAR.
```

Comment:
Nit: Would change this to say:
When set to 'False,' emissions are not generated before the simulation's start date. By default, emissions are produced leading up to this date, based on their durations, enabling the system to begin in a state without LDAR